test_scripts/Polices/App_Permissions/ATF_HP_Order_Of_Request_Response_Notification_On_Registering_App.lua

```diff
@@ -0,0 +1,138 @@
+---------------------------------------------------------------------------------------------
+-- Description:
+-- Application with appID intends to be registered on SDL, device with deviceID the application is running on is connected to HU.
+-- 1. Used preconditions:
+-- Restart SDL and clean all data to set SDL in first life cycle state.
+-- 2. Performed steps
+-- Register new application.
+--
+-- Requirement summary:
+-- The order of requests/responses/notifications during application registering must be the following:
+-- 1. app->SDL: RegisterAppInterface (policy_appID, parameters)
```

Comment:
After section Description you should have:
1. Used Preconditions 
2. Performed steps.